app/Http/Controllers/BookController.php

```diff
@@ -22,19 +22,23 @@ public function detail($slug)
     {
         try {
             $liked = false;
+            $added = false;
             $book = Book::where('slug', $slug)->firstOrFail();
             if (Auth::check()) {
-                $isLiked = $book->likedUsers()->wherePivot('user_id', Auth::id())->exists();
+                $isLiked = $book->likedUsers()->wherePivot('user_id', Auth::id());
                 if ($isLiked) {
                     $liked = true;
                 }
```

Comment:
```suggestion
                $liked = $book->likedUsers()->wherePivot('user_id', Auth::id())->exists();
```
vì exists() trả về true/false rồi nên em gán luôn cho `$liked` luôn, ko cần phải so sánh trong `if` nữa

Reply:
ahhhh vang a he he